vizro-core/src/vizro/models/_navigation/accordion.py

```diff
@@ -53,9 +53,14 @@ def build(self, *, active_page_id=None):
                     children=accordion_buttons,
                     title=page_group.upper(),
                     class_name="accordion-item-header",
+                    item_id=page_group,
                 )
             )
 
+        active_item = next(
+            (page_group for page_group, page_members in self.pages.items() if active_page_id in page_members)
+        )
```

Comment:
```suggestion
        active_item = next(
            page_group for page_group, page_members in self.pages.items() if active_page_id in page_members
        )
```

Reply:
Implemented, and it does not crash when navigating on non-existent page. Good catch. 🙌